test/e2e/network_test.go

```diff
@@ -148,6 +153,31 @@ func testEastWestNetworking(t *testing.T) {
 	}
 }
 
+// getDeploymentLogs ensures logs of multiple deployments
+// are collected via a pod labelSelector
+func (tc *testContext) getDeploymentLogs(deployments ...*appsv1.Deployment) {
+	for _, deployment := range deployments {
+		labelSelector := deployment.Name
+		tc.savePodLogs(labelSelector)
```

Comment:
No need for the temporal variable `labelSelector`.

Consider adding a comment around the `labelSelector := deployment.Name` mapping, since using a deployment name as selector is not trivial.

Reply:
I previously had "app=" as a prefix for labelSelector. I'll add it back in or revise this